tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -97,6 +97,53 @@ def setup(self, dag_maker):
 
         patch.stopall()
 
+    def test_template_fields(self):
+        assert 9 == KubernetesPodOperator.template_fields.__len__()
+        KubernetesPodOperator.template_fields.__contains__('image')
+        KubernetesPodOperator.template_fields.__contains__('cmds')
+        KubernetesPodOperator.template_fields.__contains__('arguments')
+        KubernetesPodOperator.template_fields.__contains__('env_vars')
+        KubernetesPodOperator.template_fields.__contains__('labels')
+        KubernetesPodOperator.template_fields.__contains__('config_file')
+        KubernetesPodOperator.template_fields.__contains__('pod_template_file')
+        KubernetesPodOperator.template_fields.__contains__('namespace')
+        KubernetesPodOperator.template_fields.__contains__('container_resources')
```

Comment:
These don’t do anything. Do you mean `assert "image" in KubernetesPodOperator.template_fields`?

Although honestly checking for this is pretty useless IMO and can be removed entirely.

Reply:
Hi @uranusjr.  After reviewing, I tend to agree, but do not feel strongly about it.  
I kept it in since it had been in https://github.com/nitinmuteja original PR and no one had brought anything up about it.

https://github.com/apache/airflow/pull/23531/files#diff-9eab8bfed52232ef1125033c59024d9ed878b6504b830ba633aa2ad7d1f60151

This PR (and its prior one) have been cooking for 2-3 weeks now and finally has been approved.  I can go ahead and remove these if need be, but would prefer (and commit) to doing a follow up PR to remove this if folks feel strongly about it.

Reply:
There are still some static checks failing that you need to fix so you can do it along the same fix/rebase @alanatlemba .

BTW. Just to set expectations right. This is perfectly normal that PRs take weeks, sometimes months, sometimes eve multiple months to get  reviewed and merged. This is the nature of OSS project like ours, where people review and merge commits according to when they have time, and according to priorities. This is something we explain in our contruibution guidelines - the nature of the contributions is rather asynchronous and it requires patience and persistency (a bit). We have currently 144 PRs in progress and naturally, quality and long term maintainabilitty trump speed of merging, especially when the changes affect really important and heavily used core parts of airflow and are not critical (less so when it comes to integrations and even less so when it comes to "development" tooling). Move "slow" to go "fast" is important aspect of keeping the project in the state that is maintainable.

Speed of merging individual PRs is not the "only" criteria that matters, it's actually quite secondary.

Reply:
@potiuk  I get it, and I also appreciate your response.  It's quite amazing how much is actually getting done.  So much so that I've been wondering if I'm going about it badly.  Thanks again.